meta.yaml

```diff
@@ -54,12 +55,14 @@ requirements:
     - matplotlib
     - nc-time-axis
     - netCDF4
-    - numpy
+    - numpy<=1.15.4
```

Comment:
Putting an upper limit on the version is an extreme measure, moreso when it pertains to such a fundamental package as numpy. Can you please add a convincing comment why this is needed and when it can be removed again?

Reply:
There was a `numpy` bug during the CI tests on the previous version of esmvaltool. Current version pass without problems, so the limit is removed.